app/src/features/surveys/view/survey-animals/SurveyAnimalsTable.tsx

```diff
@@ -72,16 +73,30 @@ export const SurveyAnimalsTable = ({
       )
     },
     {
-      field: 'deployments',
-      headerName: 'Device ID',
+      field: 'current_devices',
+      headerName: 'Current Devices',
```

Comment:
Nit: I think 'Active Devices' would sound more natural

Reply:
These terms come from the ticket so I think I will keep them this way unless Macgregor wants something else later.